packages/ref-core/src/cmip_ref_core/pycmec/tests/unit/test_cmec_metric.py

```diff
@@ -0,0 +1,132 @@
+from cmip_ref_core.pycmec.metric import (
+    MetricCV,
+    MetricDimensions,
+    MetricResults,
+    CMECMetric,
+)
+
+import pytest
+
+
+@pytest.fixture
+def cmec_right_metric_dict():
+    return {
+        "SCHEMA": {"name": "cmec", "version": "v1", "package": "ilamb v3"},
+        "DIMENSIONS": {
+            "json_structure": ["model", "metric"],
+            "model": {
+                "E3SM": {"name": "E3SM"},
+                "CESM2": {"name": "CESM2"},
+                "IPSL-CM5A-LR": {"name": "IPSL-CM5A-LR"},
+            },
+            "metric": {
+                "Ecosystem and Carbon Cycle": {"name": "Ecosystem and Carbon Cycle"},
```

Comment:
The contents of the metrics aren't controlled, right? i.e. you don't have a CV for valid values for the `metric` dimension

The UD just takes the dimensions as provided

Reply:
Yes. they are not controlled by CMEC. UD just reads them and plots whatever metrics in the `metric` dimension.